frontend/Dockerfile

```diff
@@ -15,18 +15,18 @@ FROM docker.io/nginx:stable-alpine
 LABEL maintainer="Chevdor <chevdor@gmail.com>"
 LABEL description="Polkadot Telemetry frontend"
 
+# When we run the built docker image, the default command uses this
+# env var to tell the UI where to look to get feed info.
 ENV SUBSTRATE_TELEMETRY_URL=
 
 WORKDIR /usr/share/nginx/html
 
-#COPY --from=builder /opt/builder/env.sh /usr/bin/
-#RUN apk add --no-cache bash; chmod +x /usr/bin/env.sh
+COPY --from=builder /opt/builder/env.sh /usr/bin/
+RUN apk add --no-cache bash; chmod +x /usr/bin/env.sh
 
-#COPY --from=builder /opt/builder/nginx/nginx.conf /etc/nginx/nginx.conf
-#COPY --from=builder /opt/builder/nginx/default /etc/nginx/sites-available/default
+COPY --from=builder /opt/builder/nginx/nginx.conf /etc/nginx/nginx.conf
 COPY --from=builder /opt/builder/build /usr/share/nginx/html
 
-EXPOSE 80
+EXPOSE 8000
```

Comment:
I can't remember why I changed the port here, but it really make sense to use a port higher than 1024 for the application inside the container. 👍

Reply:
I think it may have been like that from before your changes. I get the impression from reading the docs that `EXPOSE` doesn't really do anything useful for us, but I wasn't very clear on it. Anyway, it may have been incorrect for a while and just not had any impact on anything :)

Reply:
yeah! the expose option is only used when you run `docker -P` to publish all the ports.

Reply:
Using `EXPOSE` is also good for documentation purpose and shows what ports the container exposes.